pkg/apis/pipeline/v1alpha1/verificationpolicy_validation.go

```diff
@@ -55,6 +55,13 @@ func (vs *VerificationPolicySpec) Validate(ctx context.Context) (errs *apis.Fiel
 			errs = errs.Also(a.Key.Validate(ctx).ViaFieldIndex("key", i))
 		}
 	}
+	if vs.Mode == "" {
```

Comment:
I wouldn't require this. Just treat the empty Mode as Enforce in the reconciler, and add it to the list of allowed values below.

Reply:
This also lets you retain compatibility with existing policies.

Reply:
Yeah! That makes sense! Thanks!